core/arch/arm/kernel/generic_boot.c

```diff
@@ -78,6 +78,9 @@ KEEP_PAGER(sem_cpu_sync);
 struct dt_descriptor {
 	void *blob;
 	int frag_id;
+#ifdef CFG_OVERLAY_ADDR
+	int is_overlay;
```

Comment:
`bool`, and TBH I don't think the conditional really helps here, it makes things more complicated for little gain.